docs/configuration/config-file-reference.md

```diff
@@ -3436,6 +3441,10 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -distributor.ingestion-burst-size
 [ingestion_burst_size: <int> | default = 50000]
 
+# Per-user allowed native histogram ingestion burst size (in number of samples)
```

Comment:
Shouldn't the burst also be MaxFloat? Would it work if the burst is 0?

Reply:
Burst is int as per the Rate lib we are using for this throttling: https://github.com/cortexproject/cortex/blob/master/vendor/golang.org/x/time/rate/rate.go#L78
0 and Inf are handled as special cases in this lib: https://github.com/cortexproject/cortex/blob/master/vendor/golang.org/x/time/rate/rate.go#L37